book/src/puzzle_18/puzzle_18.md

```diff
@@ -238,32 +239,28 @@ The kernel is parameterized with:
 
 #### Shared memory allocation
 ```mojo
-shared_max = tb[dtype]().row_major[TPB]().shared().alloc()
-shared_sum = tb[dtype]().row_major[TPB]().shared().alloc()
```

Comment:
Same

Reply:
Are you suggesting to stick with the constant defined at the top which doesn't depend on the input size or to call it `TPB` instead of `num_threads`?

For me while trying to fix the puzzles TPB is the most difficult variable name to deal with because it doesn't refer to the number of threads per block it is in nearly all cases the number of threads in a given dimension.